src/assets/javascript/app/init.js

```diff
@@ -18,7 +18,7 @@ const checkForOnSubmit = document.querySelector("form[onsubmit]");
 const checkForRadio = document.querySelector('input[type="radio"]');
 let disableSubmit = false;
 
-if (checkForOnSubmit === null && checkForRadio !== null) {
+if (checkForOnSubmit === null || checkForRadio !== null) {
```

Comment:
Could we just drop this if statement completely? And let the JS manage the submit button on any form that's found?